assets/stylesheets/_vendor.scss

```diff
@@ -1,7 +1,6 @@
 // External Dependencies
 @import '../../node_modules/react-virtualized/styles';
 @import '../../node_modules/draft-js/dist/Draft';
-@import '../../node_modules/notifications-panel/src/boot/stylesheets/style';
```

Comment:
We still have `notifications-panel` as a package.json dependency but are using the Calypso notifications src styles. We'll want to make sure the other diffs are ready before we let this land, or that we complete this transition within a day or two.